<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PersonalCenter</class>
 <widget class="QWidget" name="PersonalCenter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>个人中心</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>241</width>
     <height>281</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>更改权限</string>
    </attribute>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>当前权限:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="current_permissions">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>更改权限:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="permissions">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox {
    border: 2px solid gray;
    border-radius: 3px;
    min-width: 6em;
}


QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 15px;
    border-left-width: 1px;  
	border-left-color: darkgray;
    border-left-style: solid;
    border-top-right-radius: 3px;
    border-bottom-right-radius: 3px;
}

QComboBox::down-arrow {
    image: url(:/icon/res/ui_down_arrow.png);
}

QComboBox::down-arrow:on {
    top: 1px;
    left: 1px;
}


</string>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>操作员</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>管理员</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="admin_pw">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>130</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    border: 2px solid gray;
    border-radius: 3px;
    padding: 0 8px;
    background: rgb(85, 255, 127);
    selection-background-color: darkgray;
}
</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::PasswordEchoOnEdit</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="admin_id">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    border: 2px solid gray;
    border-radius: 3px;
    padding: 0 8px;
    background: rgb(85, 255, 127);
    selection-background-color: darkgray;
}
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>输入管理员密码:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>输入管理员账号:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="confirm">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>170</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>12</pointsize>
       <weight>9</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {    
    border-radius: 7.5px;
    background-color: #00bcd4;
	font: 75 12pt &quot;黑体&quot;;
	color: white;
}

QPushButton:hover{
	background-color: #03a9f4;
	color: white;
}

QPushButton:pressed{
	background-color: #2196f3;
    border-style: inset;
}2196f3</string>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>更改密码</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>输入旧密码:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="change_pw">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>120</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>12</pointsize>
       <weight>9</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {    
    border-radius: 7.5px;
    background-color: #00bcd4;
	font: 75 12pt &quot;黑体&quot;;
	color: white;
}

QPushButton:hover{
	background-color: #03a9f4;
	color: white;
}

QPushButton:pressed{
	background-color: #2196f3;
    border-style: inset;
}2196f3</string>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="new_pw_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>90</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    border: 2px solid gray;
    border-radius: 3px;
    padding: 0 8px;
    background: rgb(85, 255, 127);
    selection-background-color: darkgray;
}
</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::PasswordEchoOnEdit</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="new_pw">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    border: 2px solid gray;
    border-radius: 3px;
    padding: 0 8px;
    background: rgb(85, 255, 127);
    selection-background-color: darkgray;
}
</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::PasswordEchoOnEdit</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="old_pw">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    border: 2px solid gray;
    border-radius: 3px;
    padding: 0 8px;
    background: rgb(85, 255, 127);
    selection-background-color: darkgray;
}
</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::PasswordEchoOnEdit</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>输入新密码:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>确认新密码:</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>current_permissions</tabstop>
  <tabstop>permissions</tabstop>
  <tabstop>admin_id</tabstop>
  <tabstop>admin_pw</tabstop>
  <tabstop>confirm</tabstop>
  <tabstop>old_pw</tabstop>
  <tabstop>new_pw</tabstop>
  <tabstop>new_pw_2</tabstop>
  <tabstop>change_pw</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
